tests/TestCase/I18n/MessagesFileLoaderTest.php

```diff
@@ -64,4 +64,15 @@ public function testCustomLocalePath()
         $messages = $package->getMessages();
         $this->assertEquals('Po (translated) from custom folder', $messages['Plural Rule 1']['_context']['']);
     }
+
+    /**
+     * Test reading MO files
+     * @return void
+     */
+    public function testMoFiles()
+    {
+        $loader = new MessagesFileLoader('empty', 'es', 'mo');
+        $package = $loader();
+        $this->assertNotNull($package);
```

Comment:
`false` is not null too. Are there any other assertions we can make about this package object?